django_heroku/core.py

```diff
@@ -43,6 +43,19 @@ def teardown_databases(self, old_config, **kwargs):
 
 
 def settings(config, *, db_colors=False, databases=True, test_runner=True, staticfiles=True, allowed_hosts=True, logging=True, secret_key=True):
+    if 'USE_MAX_CONN_AGE' in os.environ:
+        use_max_conn_age = bool(os.environ['USE_MAX_CONN_AGE'])
+    else:
+        use_max_conn_age = True
+
+    # DEVELOPMENT mode configuration.
+    ssl_required = True
+    if 'DEVELOPMENT' in os.environ:
+        logger.info('Adding $DEVELOPMENT to DEVELOPMENT Django setting.')
+        # Set the Django setting from the environment variable.
+        config['DEVELOPMENT'] = os.environ['DEVELOPMENT']
+        if os.environ['DEVELOPMENT'] == 'True':
```

Comment:
Again, many people like using `1` and `0` for boolean values. This is very limiting